apps/src/sites/studio/pages/init/loadGamelab.js

```diff
@@ -8,6 +8,10 @@ export default function loadGamelab(options) {
   options.skinsModule = skins;
   var gamelab = new GameLab();
 
+  if (IN_UNIT_TEST) {
+    window.Gamelab = gamelab;
+  }
```

Comment:
I went with having the runLevelTest file adding it to the window like @islemaster suggested. Does this seem right?